docs/dev-guide/index.md

```diff
@@ -227,13 +229,13 @@ const question = await queryRow(sql.select_question, { question_id: 45 }, Questi
 
 - For each variant of a question that a student sees they will have submitted zero or more `submissions` with a `variant_id` to show what it belongs to. The submissions row also contains information the submitted answer and whether it was correct.
 
-## DB schema (full data)
+## Database schema (full data)
 
-- See the [list of DB tables](https://github.com/PrairieLearn/PrairieLearn/blob/master/database/tables/), with the ER (entity relationship) diagram below ([PDF ER diagram](models.pdf)).
+- See the [list of database tables](https://github.com/PrairieLearn/PrairieLearn/blob/master/database/tables/), with the ER (entity relationship) diagram below:
 
-![DB Schema](models.png)
+![](./models.d2){layout="elk"}
```

Comment:
Nit: I see the layout engine is also configured in `models.d2`. Is declaring it in both places necessary?

Reply:
It looks like the d2 plugin doesn't respect the configuration in models.d2, I removed it from models.d2.